### PR TITLE
1.11

### PR DESCRIPTION
**Pull request by thisisnotashwin**
Changes proposed in this PR:
- Support Consul 1.11.0

How I've tested this PR:
- pipelines

How I expect reviewers to test this PR:
- 👀 

Checklist:
- [x] Tests added
- [x] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: charts/consul/Chart.yaml [hunk 1]
```diff
@@ -13,9 +13,9 @@ annotations:
   artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.10.4
+      image: hashicorp/consul:1.11.0
     - name: consul-k8s-control-plane
-      image: hashicorp/consul-k8s-control-plane:0.38.0
+      image: hashicorp/consul-k8s-control-plane:0.39.0
     - name: envoy
       image: envoyproxy/envoy-alpine:v1.18.4
```
Comment (david-yu) [inline charts/consul/Chart.yaml:20]:
```suggestion
      image: envoyproxy/envoy-alpine:v1.20.0
```

File: charts/consul/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: consul
-version: 0.38.0
-appVersion: 1.10.4
+version: 0.39.0
+appVersion: 1.11.0
 kubeVersion: ">=1.17.0-0"
```
Comment (david-yu) [inline charts/consul/Chart.yaml:5]:
```suggestion
kubeVersion: ">=1.18.0-0"
```

Comment (david-yu) [reply charts/consul/Chart.yaml:5]: Should be safe to bump to 1.18.0 since most K8s distros are on 1.20.x and above

Comment (david-yu) [reply charts/consul/Chart.yaml:5]: 1.18.0 is what we currently declare on the consul-k8s repo to be consistent.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ## UNRELEASED
 
 FEATURES:
+* Helm
+  * Support Consul 1.11.0. [[GH-935](https://github.com/hashicorp/consul-k8s/pull/935)]
+  * Support Envoy 1.20.0. [[GH-935](https://github.com/hashicorp/consul-k8s/pull/935)]
+  * * Minimum Kubernetes versions supported is 1.17+. [[GH-935](https://github.com/hashicorp/consul-k8s/pull/935)]
```
Comment (david-yu) [inline CHANGELOG.md:7]:
```suggestion
  * * Minimum Kubernetes versions supported is 1.18+. [[GH-935](https://github.com/hashicorp/consul-k8s/pull/935)]
```

**Comment by thisisnotashwin** [comment]
thanks @david-yu

**Review by david-yu — APPROVED** [review]

**Review by ndhanushkodi — APPROVED** [review]

**Review by t-eckert — APPROVED** [review]
LGTM

**Pull request merged**